src/modules/user/Register.jsx

```diff
@@ -0,0 +1,51 @@
+export const Register = () => {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
```

Comment:
YOu're trying to do things in the JS way, not in the react way. Use state to store the values of the input fields, then on submit use the state values. Also, here you should be using UserContext. Make sure that once the form is submitted, a corresponding dispatch action is triggered and data is sent to the context. In the reducer function in the context, you can then call whatever logic you need (`localStorage.setItem("user", JSON.stringify({ name, email }));` in this case ) from the reducer statment. 

It's important to always try to implement this separation of concerns when programming otherwise the code becomes a spagetti bowl very fast.